XamlControlsGallery/ControlPages/AutoSuggestBoxPage.xaml.cs

```diff
@@ -21,23 +21,138 @@ namespace AppUIBasics.ControlPages
+            "Dwelf",
+            "Egyptian Mau",
+            "European Shorthair",
+            "Exotic Shorthair",
+            "Foldex",
+            "German Rex",
+            "Havana Brown",
+            "Highlander",
+            "Himalayan",
+            "Japanese Bobtail",
+            "Javanese",
+            "Kanaani",
+            "Khao Manee",
+            "Kinkalow",
+            "Korat",
+            "Korean Bobtail",
+            "Korn Ja",
+            "Kurilian Bobtail",
+            "Lambkin",
+            "LaPerm",
+            "Lykoi",
+            "Maine Coon",
+            "Manx",
+            "Mekong Bobtail",
+            "Minskin",
+            "Napoleon",
+            "Munchkin",
+            "Nebelung",
+            "Norwegian Forest Cat",
+            "Ocicat",
+            "Ojos Azules",
+            "Oregon Rex",
+            "Oriental Bicolor",
+            "Oriental Longhair",
+            "Oriental Shorthair",
+            "Persian (modern)",
+            "Persian (traditional)",
+            "Peterbald",
+            "Pixie-bob",
+            "Ragamuffin",
+            "Ragdoll",
+            "Raas",
+            "Russian Blue",
+            "Russian White",
+            "Sam Sawet",
+            "Savannah",
+            "Scottish Fold",
+            "Selkirk Rex",
+            "Serengeti",
+            "Serrade Petit",
+            "Siamese",
+            "Siberian or´Siberian Forest Cat",
+            "Singapura",
+            "Snowshoe",
+            "Sokoke",
+            "Somali",
+            "Sphynx",
+            "Suphalak",
+            "Thai",
+            "Thai Lilac",
+            "Tonkinese",
+            "Toyger",
+            "Turkish Angora",
+            "Turkish Van",
+            "Turkish Vankedisi",
+            "Ukrainian Levkoy",
+            "Wila Krungthep",
+            "York Chocolate"
+        };
+
         public AutoSuggestBoxPage()
         {
             this.InitializeComponent();
         }
 
         private void AutoSuggestBox_TextChanged(AutoSuggestBox sender, AutoSuggestBoxTextChangedEventArgs args)
         {
-            if (args.Reason == AutoSuggestionBoxTextChangeReason.UserInput)
-            {
-                List<string> suggestions = new List<string>()
+            var suitableItems = new List<string>();
+            var splitText = sender.Text.ToLower().Split(" ");
+            foreach(var cat in Cats)
+            {
+                var found = splitText.All((key)=>
                 {
-                    sender.Text + "1",
-                    sender.Text + "2"
-                };
-                Control1.ItemsSource = suggestions;
+                    return cat.ToLower().Contains(key);
+                });
+                if(found)
+                {
+                    suitableItems.Add(cat);
+                }
+            }
+            if(suitableItems.Count == 0)
+            {
+                suitableItems.Add("No items found");

```

Comment:
Double checking: I would expect this text would be handled by AutoSuggestBox (so that it can be consistent, localizable, etc.) - is that not the case? If not and we need to supply our own text, please use "No results found", as this is what the main search box inside the Gallery reports.

Reply:
Unfortunately, this is something we need to handle. If the collection of items is empty, the AutoSuggestBox just shows no flyout at all. Updated the wording now to "No results found".